sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample06_Transactions.md

```diff
@@ -44,6 +44,48 @@ using (var ts = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
 }
 ```
 
+### Transactions across entities
+In the previous version of the library, when attempting to perform transactions that spanned multiple entities, it was necessary to configure your senders so that they would send-via the other entity in the transaction. With updates to the service, the new library introduces the concept of transaction groups to handle cross entity transactions. When creating senders and receivers that should be part of a cross-entity transaction, you can specify the same `TransactionGroup` string in the options as shown below. 
+
+There are a few caveats when working with transaction groups: 
+The first entity that performs an operation becomes the implicit send-via entity. This means that subsequent entities that perform their first operation need to either be senders, or if they are receivers they need to be on the same entity as the initial send-via entity. For instance, if you have SenderA and ReceiverB that are part of the same transaction group, you would need to receive first with ReceiverB to allow this to work. If you first used SenderA to send to Queue A, and then attempted to receive from Queue B, an `InvalidOperationException` would be thrown. You could still add a ReceiverA to the same transaction group after initially sending to SenderA, since they are both using the same queue. This would be useful if you also had a SenderB that you want to include as part of the transaction group (otherwise there would be no need to use transaction groups as you would be dealing with only one entity).
```

Comment:
I found this paragraph dense and am still not sure that I fully understand what you're describing or how/why things work.   For example, the initial sentence states that the first operation needs to be done by a sender, but the example in the second sentence says that `ReceiverB` needs to receive before using `SenderA`.    It's also not clear if that sender/receiver combination is using the same queue or not _(though the later sentence leads me to believe they are not here...)_

You may want to chop this up into smaller paragraphs or bullets with a single context that you specify first and then describe the interactions and caveats.